src/components/layout/CompanyClaim/CompanyClaim.js

```diff
@@ -1,38 +1,61 @@
 import React from 'react';
 // import PropTypes from 'prop-types';
-
 import styles from './CompanyClaim.module.scss';
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faMobileAlt, faShoppingBasket } from '@fortawesome/free-solid-svg-icons';
+import { useSelector } from 'react-redux';
+import { getCount } from '../../../redux/cartRedux';
 
-const CompanyClaim = () => (
-  <div className={styles.root}>
-    <div className='container'>
-      <div className='row align-items-center'>
-        <div className={`col text-left ${styles.phoneNumber}`}>
-          <p>
-            <FontAwesomeIcon className={styles.icon} icon={faMobileAlt} /> 2300 - 3560 -
-            222
-          </p>
-        </div>
-        <div className='col text-center'>
-          <a href='#'>
-            <img src='/images/logo.png' alt='Bazar' />
-          </a>
-        </div>
-        <div className={`col text-right ${styles.cart}`}>
-          <a href='#' className={styles.cartBox}>
-            <div className={styles.cartIcon}>
-              <FontAwesomeIcon className={styles.icon} icon={faShoppingBasket} />
-            </div>
-            <div className={styles.cartCounter}>0</div>
-          </a>
+const CompanyClaim = () => {
+  const cartLength = useSelector(getCount);
+
+  return (
+    <div className={styles.root}>
+      <div className='container'>
+        <div className='row align-items-center'>
+          <div className={`col text-left ${styles.phoneNumber}`}>
+            <p>
+              <FontAwesomeIcon className={styles.icon} icon={faMobileAlt} /> 2300 - 3560
+              - 222
+            </p>
+          </div>
+          <div className='col text-center'>
+            <a href='#'>
+              <img src='/images/logo.png' alt='Bazar' />
+            </a>
+          </div>
+          <div className={`col text-right ${styles.cart}`}>
+            <a href='#' className={styles.cartBox}>
+              <div className={styles.cartIcon}>
+                <FontAwesomeIcon className={styles.icon} icon={faShoppingBasket} />
+              </div>
+              <div className={styles.cartCounter}>
+                <span
+                  className={
+                    cartLength >= 99999 || cartLength <= 99999
```

Comment:
Takiej ifologi nie możemy robić, to zadanie można poprawić jedynie przy zmianie CSS dla klasy koszyka która już była ;)